app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt

```diff
@@ -113,6 +118,24 @@ class AdministratorControlsActivityTest {
   @Inject
   lateinit var context: Context
 
+  @get:Rule
+  val activityTestRule = ActivityTestRule(
+    AdministratorControlsActivity::class.java,
+    /* initialTouchMode= */ true,
+    /* launchActivity= */ false
+  )
+
+  @Test
+  fun testAdministratorControls_hasCorrectActivityLabel() {
```

Comment:
please add a screenshot of the espresso test result showing this test case is passing.

Reply:
![RunTest](https://user-images.githubusercontent.com/42642325/116805186-3f1d2d00-ab42-11eb-8100-6900307e50c2.png)

Reply:
Could you add this screenshot to the PR description.

Reply:
Added screenshot to the PR description.